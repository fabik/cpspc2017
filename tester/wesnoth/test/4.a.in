100000 40
1
39935
