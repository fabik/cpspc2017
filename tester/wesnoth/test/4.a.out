39935 100000 0.6637150759
